client_perms/client.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+# enable some more output
+set -x
+
+[ -n "${SERVER}" ]
+[ -n "${EXPORT}" ]
+
+if [ "$1" = "client_initialization" ]
+then
+	# install build and runtime dependencies
+	yum -y install nfs-utils time
+
+	mkdir -p /mnt/ganesha
+
+	mount -t nfs -o vers=3 ${SERVER}:${EXPORT} /mnt/ganesha
+
+	echo "Client Initial Stage --- With All Rights To All Clients ( RO & RW ) "
+
+	cd /mnt/ganesha
+	
+	echo "Trying To Write A File"
+	echo "Hello World" > testFile.txt
+	ret=$?
+	if [ $ret -eq 0 ]
+	then
+		echo "SUCCESS"
+	else
+		echo "FAILED ON WRITING RIGHTS"
+		#exit ret
```

Comment:
How do these indicate failure to the test runner?  Is the output parsed?

Reply:
@dang 
Sorry, The comment line `#exit ret` should be uncommentd ... :(